http/http_client.hpp

```diff
@@ -206,75 +217,87 @@ class HttpClient : public std::enable_shared_from_this<HttpClient>
     {
         state = ConnState::recvInProgress;
 
-        auto respHandler = [self(shared_from_this())](
-                               const boost::beast::error_code ec,
-                               const std::size_t& bytesTransferred) {
-            if (ec && ec != boost::asio::ssl::error::stream_truncated)
-            {
-                BMCWEB_LOG_ERROR << "recvMessage() failed: " << ec.message()
-                                 << " from subId: " << self->subId;
-                self->state = ConnState::recvFailed;
-                self->handleConnState();
-                return;
-            }
+        auto respHandler =
+            [self(shared_from_this())](const boost::beast::error_code ec,
+                                       const std::size_t& bytesTransferred) {
+                std::string_view eventStr = self->req.body();
+                nlohmann::json event;
+                auto eventData = event.parse(eventStr);
+                if (ec && ec != boost::asio::ssl::error::stream_truncated)
```

Comment:
We just keep going when it is this ec, boost::asio::ssl::error::stream_truncated ?

Reply:
Yes. This was there earlier as well. I just enhanced the error trace